src/main/java/org/folio/service/orders/lines/update/OrderLinePatchOperationService.java

```diff
@@ -27,38 +54,109 @@ public OrderLinePatchOperationService(RestClient restClient, OrderLinePatchOpera
     this.purchaseOrderLineService = purchaseOrderLineService;
   }
 
-  public CompletableFuture<Void> patch(String lineId, PatchOrderLineRequest request, RequestContext requestContext){
+  public CompletableFuture<Void> patch(String lineId, PatchOrderLineRequest request, RequestContext requestContext) {
+    return patchOrderLine(request, lineId, requestContext)
+      .thenCompose(v -> updateInventoryInstanceInformation(request, lineId, requestContext));
+  }
+
+  private CompletableFuture<Void> patchOrderLine(PatchOrderLineRequest request,
+                                                 String lineId, RequestContext requestContext) {
+    CompletableFuture<Void> future = new CompletableFuture<>();
+
+    purchaseOrderLineService.getOrderLineById(lineId, requestContext)
+      .thenAccept(poLine -> {
+        OrderLineUpdateInstanceHolder orderLineUpdateInstanceHolder = new OrderLineUpdateInstanceHolder()
+          .withPathOrderLineRequest(request)
+          .withStoragePoLine(poLine);
```

Comment:
Can we set poLine with all required Inventory fields here before patch request?
In this case we will send only 1 REST request to update poLine instead of 2 that we have now

Reply:
The poLine required only for initializing updateInstanceHolder, it is not included into patch request payload.